watir-rspec.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = Watir::RSpec::VERSION
 
-  gem.add_dependency "rspec", "~>2.0"
+  gem.add_dependency "rspec", "~>3.0"
```

Comment:
I didn't  try running the new version with RSpec 2.X.
However I assume it will not work as  the HTMLFormatter is now adjusted to work with RSpec v 3 (it uses a ::RSpec::Core::Formatters.register which is not available in version 2.99)